java/Stdin.java

```diff
@@ -0,0 +1,24 @@
+// Test script to get input, transform, and write to stdout
+
+import java.util.Scanner;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class Stdin{
+    public static void main(String[] args){
+        Scanner scanner = new Scanner(System.in);
+        AtomicInteger counter = new AtomicInteger(1);
+        String line;
+
+        do {
+            line = scanner.nextLine();
+            if (!line.isEmpty()) { // Skip processing if the line is empty
+                System.out.println(counter.getAndIncrement() + " " + line.toUpperCase());
+            }
+        } while (scanner.hasNextLine());
+
+        scanner.close();
```

Comment:
Did [ChatGPT's suggestion](https://chatgpt.com/share/7abfc7a6-6a58-4596-9fc2-90024ce5b01e) (the last code block in that chat) not work out?

Reply:
The original solution was ignoring the last line in the input. I changed it again so that it is using stream but no longer missing the last line either.

Reply:
Great job.

Sorry, I asked because github was showing me an outdated version of the file 🙄, but now I see the version you updated it to.